docs/variants/protectli_vp66xx/building-manual.md

```diff
@@ -0,0 +1,50 @@
+# Building manual
```

Comment:
As above.

Reply:
Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4